mockgcp/mocksql/normalize.go

```diff
@@ -23,7 +25,29 @@ var _ mockgcpregistry.SupportsNormalization = &MockService{}
 func (s *MockService) ConfigureVisitor(url string, replacements mockgcpregistry.NormalizingVisitor) {
 	// SQLUser
 	replacements.ReplacePath(".items[].passwordPolicy.status.passwordExpirationTime", "2025-06-19T01:02:03Z")
+
+	// SQLInstance
+	replacements.ReplacePath(".etag", "abcdef0123A=")
+	replacements.ReplacePath(".createTime", "2024-04-01T12:34:56.123456Z")
+	replacements.ReplacePath(".ipAddresses[].ipAddress", "10.1.2.3")
+	replacements.ReplacePath(".serverCaCert.createTime", "2024-04-01T12:34:56.123456Z")
+	replacements.ReplacePath(".serverCaCert.expirationTime", "2024-04-01T12:34:56.123456Z")
+	replacements.ReplacePath(".serverCaCert.sha1Fingerprint", "12345678")
+	replacements.ReplacePath(".serverCaCert.commonName", "common-name")
+	replacements.ReplacePath(".settings.settingsVersion", "123")
+
+	// SQL Operation
+	replacements.ReplacePath(".insertTime", "2024-04-01T12:34:56.123456Z")
+	replacements.ReplacePath(".startTime", "2024-04-01T12:34:56.123456Z")
+	replacements.ReplacePath(".endTime", "2024-04-01T12:34:56.123456Z")
 }
 
 func (s *MockService) Previsit(event mockgcpregistry.Event, replacements mockgcpregistry.NormalizingVisitor) {
+	event.VisitResponseStringValues(func(path string, value string) {
+		if path == ".serviceAccountEmailAddress" {
```

Comment:
Is the abcdef some sort of random value?   I would have guessed this would be a P4SA, which we should handle purely by replacing the projectNumber, so a comment would be helpful here!